easybuild/easyconfigs/p/Porechop/Porechop-0.2.3-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'PythonPackage'
+
+name = 'Porechop'
+version = '0.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/rrwick/Porechop'
+description = """Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
+ Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,
+ it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments
+ to effectively find adapters, even at low sequence identity"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/rrwick/Porechop/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['bfed39f82abc54f44fffd9b13d2121868084da7ac3d158ac9b9aa6fa0257f0f4']
+
+dependencies = [
+    ('Python', '3.5.2'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/porechop', 'lib/python%(pyshortver)s/site-packages/porechop/porechop.py'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/porechop-%(version)s-py%(pyshortver)s.egg-info']
```

Comment:
@pescobar We usually just check for a non-empty `'lib/python%(pyshortver)s/site-packages'` dir, since the `.egg` stuff may be either a file or a directory, depends on the context...